src/components/Experience.svelte

```diff
@@ -13,10 +16,20 @@
   export let currentIndex: number
 
   let events: Event[]
+  let userDoc: DocumentReference<DocumentData>
+  api.currentUserDoc((v) => { userDoc = v })
+
+  const sendResults = () => {
+    const experimentResult: Experiment = {
+      id: `${displayChunkLength}-${inputChunkLength}`,
+      events
+    }
+    api.addExperimentRequest(userDoc, experimentResult)
+  }
 
   const onFilled = (): void => {
     $swiper.slideNext()
-    // TODO: Store events in firestore
+    sendResults()
```

Comment:
♻️ This component shouldn't send to database but the parent.

Reply:
✅